package.json

```diff
@@ -30,11 +30,13 @@
   },
   "homepage": "https://github.com/asyncapi/go-template#readme",
   "dependencies": {
+    "@asyncapi/generator": "^1.8.0",
+    "@asyncapi/parser": "^1.6.0",
```

Comment:
they are for sure not needed, actually sometimes in `devDependencies` for integration tests